src/test/resources/invalidUpsertInput9.jsonl

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "v1",
+    "values": [
+      3,
+      2,
+      1
+    ],
+    "sparse_values": {
+      "indices": [
+        4294967296
```

Comment:
should fail for 4294967296 since the range for unsigned indices is [0, 4294967295]